src/app/pages/token/token-list/token-cw20/token-cw20.component.html

```diff
@@ -184,22 +184,25 @@ <h1 class="mb-0 text--white">Fungible Tokens</h1>
             <ng-container matColumnDef="inChainValue">
               <th mat-header-cell *matHeaderCellDef>In-Chain Supply Value</th>
               <td mat-cell *matCellDef="let element" class="vertical-align-top py-4">
-                <ng-container *ngIf="element.inChainValue >= 0; else noData">
+                <ng-container *ngIf="element.price > 0; else zeroValue">
                   <div class="aura-tooltip-contain">
                     <div class="aura-tooltip-object">
                       <div class="text--gray-1 item-amount">
-                        <span *ngIf="element.inChainValue | lt: 0.01">
-                          {{ element.inChainValue == 0 ? '$0.00' : '< $0.01' }}
+                        <span *ngIf="element.price * element.totalSupply | lt: 0.01">
+                          {{ element.price == 0 ? '$0.00' : '< $0.01' }}
                         </span>
-                        <div *ngIf="element.inChainValue | gte: 0.01">
-                          $<span [decimal]="0" [appBigNumber]="(element.inChainValue || 0).toFixed(2)"></span>
+                        <div *ngIf="element.price * element.totalSupply | gte: 0.01">
+                          <span
+                            [decimal]="0"
+                            [appBigNumber]="element.totalSupply"
+                            [tokenPrice]="element.price"></span>
                         </div>
                       </div>
                     </div>
                     <app-tooltip-customize
                       *ngIf="
-                        (element.inChainValue | gte: 1000000) ||
-                        ((element.inChainValue | lt: 0.01) && element.inChainValue != 0)
+                        (element.price * element.totalSupply | lt: 0.01) ||
```

Comment:
Chỗ nhân này `totalSupply ` lớn quá có thể sai nhé. Sử dụng pipe balance
```suggestion
                        (element.totalSupply| balance: 0: true).multipliedBy(element.price ).lt(0.01)
```